### PR TITLE
Fix broken contributing links

### LINKED ISSUE #1816: Contribute link not working
Tried to click on contribute it doesnt show a correct page.
image

### PR DESCRIPTION
**Pull request by arusahni**
# Description

This fixes the outdated/broken contribution link on the index pages.

Fixes #1816 

## Status
**READY**

## Type of change

- Documentation (fix or adds documentation)

# How Has This Been Tested?

-  I viewed the rendered Markdown

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ See [our full documentation](https://docs.opsdroid.dev) to get started.
 
 ### Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](https://docs.opsdroid.dev/en/stable/contributing.html#contributing)].
+This project exists thanks to all the people who contribute. [[Contribute](https://docs.opsdroid.dev/en/stable/contributing/index.html#contributing)].
```
Comment (jacobtomlinson) [inline README.md:41]:
```suggestion
This project exists thanks to all the people who contribute. [[Contribute](https://docs.opsdroid.dev/en/stable/contributing/)].
```

File: docs/index.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Help support opsdroid in one click by pressing [![Tweet](https://img.shields.io/
 
 ### Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](https://docs.opsdroid.dev/en/stable/contributing.html#contributing)].
+This project exists thanks to all the people who contribute. [[Contribute](https://docs.opsdroid.dev/en/stable/contributing/index.html#contributing)].
```
Comment (jacobtomlinson) [inline docs/index.md:86]:
```suggestion
This project exists thanks to all the people who contribute. [[Contribute](https://docs.opsdroid.dev/en/stable/contributing/)].
```

**Review by jacobtomlinson — CHANGES_REQUESTED** [review]
Thanks for this. I don't think we need the `index.html` or title anchor in these links.

**Comment by jacobtomlinson** [comment]
There is also a conflict because one of these links was updated in #1818. Could you please resolve this.

File: README.md (hunk 1 above)
Comment (arusahni) [reply README.md]: Done.

File: docs/index.md (hunk 2 above)
Comment (arusahni) [reply docs/index.md]: Done!

**Comment by arusahni** [comment]
@jacobtomlinson I rebased on master and resolved the conflict. LMK if you'd like to see anything else.

**Review by cognifloyd — APPROVED** [review]

**Review by jacobtomlinson — APPROVED** [review]

**Comment by jacobtomlinson** [comment]
Thanks @arusahni

**Pull request merged**